src/PanelTraits/Read.php

```diff
@@ -207,6 +213,26 @@ public function getCellView($column, $entry)
         }
     }
 
+    /**
+     * Add field/column name to include in select query when using ajax tables
+     *
+     * @param string $field field/column name
+     */
+    public function addAjaxExtraField($field)
```

Comment:
Is this adding to some other non-merged pull - the pull description says something about Blade comments and this is another function...